database/Employee.sql

```diff
@@ -0,0 +1,22 @@
+SHOW DATABASES;
+
+USE AgileSprints_SamB;
```

Comment:
This script shouldn't assume that the DB exists as we want to be able to run it to create the DB from scratch. Lets add a 'create if not exists' statement.